src/picongpu/CMakeLists.txt

```diff
@@ -268,6 +268,18 @@ if( (Boost_VERSION EQUAL 105500) AND
       "${CUDA_NVCC_FLAGS} \"-DBOOST_NOINLINE=__attribute__((noinline))\" ")
 endif()
 
+################################################################################
+# OPENMP
+##############################################################################
+
+FIND_PACKAGE(OpenMP)
+IF(NOT OPENMP_FOUND)
+    MESSAGE(STATUS "Disable OpenMP: Can't find library")
+
+ELSE()
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
```

Comment:
this is a copy-paste from a stackoverflow list. one should use `add_definitions()` instead.